proposals/0021-vk-coop-matrix.md

```diff
@@ -0,0 +1,346 @@
+<!-- {% raw %} -->
+
+# Vulkan cooperative matrix extension
+
+*   Proposal: [0021](0021-vk-coop-matrix.md)
+*   Author(s): [Steven Perron](https://github.com/s-perron)
+*   Sponsor: TBD
+*   Status: **Accepted**
+*   Required Version: Vulkan 1.1
```

Comment:
SPIR-V spec requires SPIR-V 1.6, so seems like Vulkan 1.3 should be the required version as older version don't require SPIR-V 1.6 support. (even if the VK extension has no min VK version requirement as long as VK_KHR_get_physical_device_properties2 is present)